README.spanish.md

```diff
@@ -837,7 +886,7 @@ Todo lo de encima hubiera devuelto `false` si se hubiera usado `===`
 
 **De lo contrario:** Los tokens caducados o extraviados pueden ser utilizados maliciosamente por un tercero para acceder a una aplicación y hacerse pasar por el propietario del token.
 
-🔗 [**Leer más: Blacklist JSON Web Tokens**](/sections/security/expirejwt.spanish.md)
+🔗 [**Leer más: Bloquea JSON Web Tokens caducados**](./sections/security/expirejwt.spanish.md)
```

Comment:
We dont have a literal translation for the **action** of blacklisting someone.
Said that cause the translation would be "lista negra" but that is the noun, not the action